plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergColumnHandle.java

```diff
@@ -184,4 +186,35 @@ public String toString()
     {
         return getId() + ":" + getName() + ":" + type.getDisplayName();
     }
+
+    public static IcebergColumnHandle pathColumnHandle()
+    {
+        IcebergColumnHandle icebergColumnHandle = new IcebergColumnHandle(
+                columIdentity(FILE_PATH),
+                FILE_PATH.type(),
+                ImmutableList.of(),
+                FILE_PATH.type(),
+                Optional.empty());
+
+        return icebergColumnHandle;
+    }
+
+    public static ColumnMetadata pathColumnMetadata()
+    {
+        return ColumnMetadata.builder()
+                .setName(FILE_PATH.columnName())
+                .setType(FILE_PATH.type())
+                .setHidden(true)
+                .build();
+    }
+
+    private static ColumnIdentity columIdentity(IcebergMetadataColumn metadata)
+    {
+        return new ColumnIdentity(metadata.id(), metadata.columnName(), metadata.typeCategory(), ImmutableList.of());
+    }
```

Comment:
You could even inline these, since they're only used the one time each.

Reply:
fair point, though keeping it inline, was a bit more verbose, as using the `FILE_PATH` variable multiple times.
So, as we discussed as well, this looks good to me, I hope that is fine. thanks